docs/api_walkthrough.rst

```diff
@@ -28,10 +28,10 @@ If this returns the same address, we know that the Raiden node is up and running
 .. _bootstrapping-a-token-network:
 
 Bootstrapping a token network
-=============================
-In this scenario it is assumed that a user holds some ERC20 tokens of a type that has not yet been registered in the Raiden smart contracts. Assume that the address of the token is ``0x9aBa529db3FF2D8409A1da4C9eB148879b046700``.
+===============================
+In this scenario it is assumed that a user holds some ERC20 tokens of a type that has not yet been registered in the Raiden smart contract with address ``0x9aBa529db3FF2D8409A1da4C9eB148879b046700``.
```

Comment:
To me it sounds as if the smart contract has address ``0x9aBa....``, but it is the token that has that address. Can you make this more clear?